runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MetricsContainerStepMap.java

```diff
@@ -77,6 +77,25 @@ public void update(String step, MetricsContainerImpl container) {
     getContainer(step).update(container);
   }
 
+  @Override
```

Comment:
I'm just curious as to what this change is for? Is this now being shuffled and wasn't previously?

Reply:
Good question ! Actually, I did this change because the test involved a comparison (where it wasn't the case previously).